index.html

```diff
@@ -269,5 +254,61 @@ <h3 class="about-title">Skills</h3>
           </div>
         </div>
       </section>
+
+      <section id="contact-form">
+        <div class="contact-form">
+
+          <div class="title">
+           <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+          </div>
+
+          <form action="https://formspree.io/f/xwkjdndo" class="contact-container" method="post">
+            <ul>
+              <li class="contact-full-name">
+                <label for="full-name" class="label">Full name:</label>
+                <input type="text" id="full-name" name="full-name" placeholder="Full name" maxlength="30" required>
+              </li>
+              <li class="contact-email">
+                <label for="email" class="label">Email address:</label>
+                <input type="text" id="email" name="email" placeholder="Email address" required>
+              </li>
+              <li class="contact-message">
+                <label for="message" class="label">Your message:</label>
+                <li class="contact__form__element message">
+                  <label class="label" for="message">Your message:</label>
```

Comment:
- [ ] Kindly consider using the `aria-label` attribute instead of having a `label` tag that has text but displays it as `none` because it doesn't adhere to your design template. The `aria-label` attribute provides a text alternative for screen reader users to understand the purpose of the input element. Therefore, in this case, it would be more appropriate to use the `aria-label` attribute on the input element instead of a `label` and displaying it as `none.`